src/actions/mod.rs

```diff
@@ -98,13 +98,15 @@ impl ActionHandler {
                                                   project_path: &Path)
             -> Vec<NotificationMessage<PublishRustDiagnosticsParams>>
         {
+            let cwd = ::std::env::current_dir().unwrap();
+
             build_results
             .iter()
             .map(|(path, diagnostics)| {
                 let method = "textDocument/publishDiagnostics".to_string();
 
                 let params = PublishRustDiagnosticsParams {
-                    uri: Url::from_file_path(project_path.join(path)).unwrap(),
+                    uri: Url::from_file_path(cwd.join(path)).unwrap(),
```

Comment:
This might be obvious, but I'm blanking - how do  we know the cwd will always be where the project_path is?  What if the RLS is started in one directory, VSCode is started in a second directory, and the project is a third directory?